puppet/modules/quickstack/manifests/neutron/all.pp

```diff
@@ -130,6 +130,14 @@
   }
   File['/etc/neutron/plugin.ini'] -> Exec['neutron-db-manage upgrade']
 
+  # short-term workaround for BZ 1181592
+  package{'bind-utils': } ->
+  exec {'neutron-etc-hosts-workaround':
+    command => 'dig A $(hostname) | grep -A1 "ANSWER SEC" | tail -n 1 | awk \'{print $NF " " $1}\' | sed -e \'s/.$//g\' >>/etc/hosts',
```

Comment:
If the $(hostname) is not resolvable, /etc/hosts does not get updated during the puppet run.  However, it would get updated in subsequent puppet runs if it does become resolvable.

Is that acceptable or do we need another way of modifying /etc/hosts if $(hostname) is not resolvable?